suite-native/assets/src/components/AssetItem.tsx

```diff
@@ -16,6 +16,8 @@ type AssetItemProps = {
     iconName: CryptoIconName;
     onPress?: () => void;
     fiatBalance: string;
+    fiatPercentual: number;
```

Comment:
nit pick: I don't have strong opinion on this but for simplicity I would probably prefer to use just `percentage` everywhere instead of `percentual` even if it's gramatically correct.